Pinta.Core/Managers/WorkspaceManager.cs

```diff
@@ -44,66 +45,63 @@ public interface IWorkspaceService
 		SelectionModeHandler SelectionHandler { get; }
 	}
 
-	public class WorkspaceManager : IWorkspaceService
+	public sealed class WorkspaceManager : IWorkspaceService
 	{
 		private int active_document_index = -1;
 		private int new_file_name = 1;
 
 		public WorkspaceManager ()
 		{
-			OpenDocuments = new List<Document> ();
+			var openDocumentsBacking = new List<Document> ();
```

Comment:
I feel like this is adding more complexity for someone reading the code. Can we remove the temporary `openDocumentsBacking` and rename `open_documents_backing` to just `open_documents`?
(Similar for the other classes that were modified)

Reply:
Yes, it's those pesky line endings. I've now reduced the annoyance to the minimum, though, only one line that for some reason was not normalized remains